cucumber/policy/features/replace.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Replacing a policy
+
+A policy can be reloaded using the --replace flag
+
+  Scenario: A multifile policy with one modified file fails on reload
+
+    Given I replace the policy by loading the policy file "multifile1.yml"
```

Comment:
To me, moving the policies to files makes it harder to understand what the cucumber test is doing. Can we discuss putting the policies in the cucumber code itself, like the other tests?

Reply:
I agree that putting the policy in the test would be more consistent with the other tests in this suite. However, the policies I was working with when I got the tests up were long enough that it seemed less neat to put them directly in the test file. Creating a policy directory is consistent with what's been done on other projects for longer policy files, and I tried to name them in a way that it would be clear which files you were looking for if you hopped into that directory to explore.

I'm open to revising this if you think it's important, but I can imagine future tests that we might add where I'd also want to put the policy in a separate file.